### PR TITLE
Fix for SDW Deposit error messages

### PR DESCRIPTION
**Pull request by mwodahl**
Updates the default SDX url and adds a pause in-between sending messages to prevent overloading the SDX.

### CODE REVIEW

File: src/main/java/jpo/sdw/depositor/DepositorProperties.java [hunk 1]
```diff
@@ -20,7 +20,7 @@ public class DepositorProperties implements EnvironmentAware {
 
    private static final String DEFAULT_GROUP_ID = "usdot.jpo.sdw";
    private static final String DEFAULT_KAFKA_PORT = "9092";
-   private static final String DEFAULT_DESTINATION_URL = "https://sdx-service.trihydro.com/api/deposit";
+   private static final String DEFAULT_DESTINATION_URL = "https://sdxdev-service.trihydro.com/api/deposit-multi";
```
Comment (payneBrandon) [inline src/main/java/jpo/sdw/depositor/DepositorProperties.java]: We'll want this default to point to the production service rather than dev

**Review by payneBrandon — CHANGES_REQUESTED** [review]
We talked about this via chat, but just to summarize here as well - we'll want to check into the errors on the SDX side to ensure we can process large chunks of data. I think on this depositor we'd want to limit the number of messages in a deposit and loop over it rather than introduce a delay here, which may affect lag.

File: src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java [hunk 2]
```diff
@@ -56,6 +58,12 @@ public void run(String... topics) {
             this.jsonMsgList.put("depositRequests", jsonRequests);
             this.getRestDepositor().deposit(jsonMsgList.toString());
          }
+         try {
+            // add 1 second sleep between sending messages to SDX
+            TimeUnit.SECONDS.sleep(1);
```
Comment (dmccoystephenson) [inline src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java]: Looks like Brandon suggested moving away from this delay & implementing a batch processing solution instead.

**Review by dmccoystephenson — CHANGES_REQUESTED** [review]

File: src/main/java/jpo/sdw/depositor/DepositorProperties.java (hunk 1 above)
Comment (mwodahl) [reply src/main/java/jpo/sdw/depositor/DepositorProperties.java]: Changed this to point back to the production service URL

File: src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java (hunk 2 above)
Comment (mwodahl) [reply src/main/java/jpo/sdw/depositor/consumerdepositors/KafkaConsumerRestDepositor.java]: This has been removed

**Comment by mwodahl** [comment]
The 500 errors on the SDX side have been resolved by Michael's SDX bug fix. I did modify KafkaConsumerRestDepositor.java to copy individual messages to avoid a list of duplicate message entries.

**Review by dmccoystephenson — APPROVED** [review]
Looks good!

File: src/main/java/jpo/sdw/depositor/DepositorProperties.java (hunk 1 above)
Comment (payneBrandon) [reply src/main/java/jpo/sdw/depositor/DepositorProperties.java]: We'll still want this pointing to the /deposit-multi endpoint to take advantage of the multiple object deposit. We'll just want it to target the production service: https://sdx-service.trihydro.com/api/deposit-multi

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one comment about the deposit-multi endpoint

File: src/main/java/jpo/sdw/depositor/DepositorProperties.java (hunk 1 above)
Comment (mwodahl) [reply src/main/java/jpo/sdw/depositor/DepositorProperties.java]: Updated this to target the production deposit-multi endpoint

**Review by payneBrandon — APPROVED** [review]

**Pull request merged**